src/main/java/com/wcc/platform/domain/cms/attributes/Mentor.java

```diff
@@ -0,0 +1,6 @@
+package com.wcc.platform.domain.cms.attributes;
+
+import jakarta.validation.constraints.NotEmpty;
+
+/** Mentor section to be included in the Study Group card. */
+public record Mentor(@NotEmpty String title, @NotEmpty String name, @NotEmpty String uri) {}
```

Comment:
you can remove this class and only use `StudyGroup` in my opinion it will work very well.